### ISSUE TITLE
Datovælger - Missing form label (Lever ikke op til WCAG 2)

### ISSUE THREAD

**Issue by emilnygaardfrisk** [detfaellesdesignsystem/dkfds-components#242]
[Indmeld kun 1 fejl pr. issue og tjek gerne om fejlen allerede er indmeldt. (Only one bug per issue and make sure to check if the issue already exists)]

**Bekskriv fejlen (Describe the bug)**
Det interne inputfelt for datovælgeren har ikke noget label. For a leve op til WCAG 2, skal inputfeltet enten have et label eller være type hidden.

Se referencer:
[1.1.1 Non-text Content (Level A)](https://webaim.org/standards/wcag/checklist#sc1.1.1)
[1.3.1 Info and Relationships (Level A)](https://webaim.org/standards/wcag/checklist#sc1.3.1)
[2.4.6 Headings and Labels (Level AA)](https://webaim.org/standards/wcag/checklist#sc2.4.6)
[3.3.2 Labels or Instructions (Level A)](https://webaim.org/standards/wcag/checklist#sc3.3.2)

**Genskab fejlen (To Reproduce)**
1. Gå til https://designsystem.dk/komponenter/datovaelger/
2. Åben WAVE Web Accessibility Evaluation Tool
3. Se fejlen

**Indsæt link (Add a URL)**
https://designsystem.dk/komponenter/datovaelger/

**Screenshots**
![image](https://github.com/user-attachments/assets/e2529bce-2aee-477b-87ff-7bdd32b0aef8)
![image](https://github.com/user-attachments/assets/b450b230-eceb-43f1-9ad8-7bd13f56dc2d)

**Sådan burde det fungere (Expected behavior)**
Det interne inputfelt skal være `type=hidden`

**Desktop - udfyld venligst følgende (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome
 - Version: 127.0.6533.100

**Comment by ClausWiingreen** [comment]
Er det muligt at gå over til native date picker i stedet? Det vil løse WCAG problemerne men til gengæld er det browseren der styre hvordan date pickeren ser ud (hvilket kan være en fordel).

Selve date pickeren vil ændre en smule udseen, men input feltet kommer til at se sådan ud:

<img width="244" height="88" alt="Image" src="https://github.com/user-attachments/assets/af86494e-046d-4f21-86ab-ceca424c48fc" />

Før:

<img width="255" height="87" alt="Image" src="https://github.com/user-attachments/assets/32ee3c07-44d3-466f-a647-95ad4fc26695" />

Hvilket er meget tæt på hvad der er i dag, det er bare pickeren der bliver anderledes. Her ses chrome:

<img width="238" height="334" alt="Image" src="https://github.com/user-attachments/assets/dc52355b-3d85-4ec0-be28-677c66b784ff" />

Firefox med mørkt OS tema:
<img width="295" height="386" alt="Image" src="https://github.com/user-attachments/assets/13a33a3f-4245-479c-b164-b07799598d3b" />

Før:

<img width="374" height="383" alt="Image" src="https://github.com/user-attachments/assets/e8c302fc-791e-4dcd-8d84-1ef834408f28" />

Tilgængæld behøver man ikke længere initialisere komponentet med JavaScript.

## Opsummering
* Native date picker løser WCAG
* Det vil være en breaking change da man skal ændre type="text" til "date" og javascript initializering ikke vil være nødvendigt
* Man giver browser ansvar for at style date pickeren
   * Det betyder at den måske ikke følger andet design fra design systemet
   * Men det vil passe bedre til brugerens forventninger fra operativ systemet

**Comment by detfaellesdesignsystem** [comment]
Hej @ClausWiingreen 
 
Tak for din henvendelse.
Vi er opmærksomme på, at vores nuværende datovælger har nogle udfordringer. Vi er derfor i gang med at udvikle en ny og forbedret datovælger, som vil udkomme med næste major release.
 
Venlig hilsen
Det Fælles Designsystem
